code/dynamic_programming/egg_dropping_puzzle/eggdrop.c

```diff
@@ -0,0 +1,56 @@
+#include<stdio.h>
+#include<limits.h>
+int max(int a,int b)
+{
+  if(a>b)
+  return a;
+  else
+  return b;
+}
+  
+int eggdrop(int n,int k)
+{
+ int i,j,x;
+ int table[n+1][k+1];
```

Comment:
Your code is sometimes indented with 1 space and other times with 2 spaces. I believe it would be better to unify the indentation with 4 spaces, as it is more usual, to improve the readability.